packages/babel-parser/src/parser/expression.js

```diff
@@ -765,12 +765,26 @@ export default class ExpressionParser extends LValParser {
     optional: boolean,
   ): N.Expression {
     if (node.callee.type === "Import") {
-      if (node.arguments.length !== 1) {
-        this.raise(node.start, Errors.ImportCallArity);
+      if (node.arguments.length === 2) {
+        this.expectPlugin("moduleAttributes");
+      }
+      if (node.arguments.length === 0 || node.arguments.length > 2) {
+        this.raise(
+          node.start,
+          Errors.ImportCallArity,
+          this.hasPlugin("moduleAttributes")
+            ? "one or two arguments"
+            : "one argument",
+        );
       } else {
-        const importArg = node.arguments[0];
-        if (importArg?.type === "SpreadElement") {
-          this.raise(importArg.start, Errors.ImportCallSpreadArgument);
+        const importArgSpreadElement = node.arguments.find(
+          arg => arg.type === "SpreadElement",
+        );
+        if (importArgSpreadElement) {
+          this.raise(
+            importArgSpreadElement.start,
+            Errors.ImportCallSpreadArgument,
+          );
         }
```

Comment:
This is a minor point, but if we have two spreads (`import(...a, ...b)`) is should report both.

```suggestion
        for (const arg of node.arguments) {
          if (arg.type === "SpreadElement") {
            this.raise(arg.start, Errors.ImportCallSpreadArgument);
          }
        }
```